src/metpy/plots/skewt.py

```diff
@@ -600,10 +627,14 @@ def plot_mixing_lines(self, w=None, p=None, **kwargs):
 
         # Set pressure range if necessary
         if p is None:
-            p = np.linspace(600, max(self.ax.get_ylim())) * units.mbar
+            # from highest pressure to 60% of highest pressure
+            pmax = max(self.ax.get_ylim())
+            p = np.linspace(0.6 * pmax, pmax) * self.ax.yaxis.units
```

Comment:
Could we leave as 600 hPa, but make it express in proper units as:
```python
pmin = units.Quantity(600, 'hPa').to(self.ax.yaxis.units)
```
?

Reply:
Sure thing.